pages/ja/lb4/Not-yet-translated.md

```diff
@@ -1,10 +0,0 @@
----
-title: "Not yet translated"
```

Comment:
@nabdelgadir, I think we need this page. Could you please confirm? Thanks.

Reply:
Yes, we still need this page for other pages that aren't translated